src/apps/aave/positions.ts

```diff
@@ -111,211 +150,152 @@ const hook: PositionsHook = {
+              address: reserveData.aTokenAddress.toLowerCase(),
+              tokens: [
                 {
-                  percentage: supplyApy,
-                  label: t('yieldRates.earningsApy'),
-                  tokenId: getTokenId({
-                    networkId,
-                    address: reserveData.underlyingAsset.toLowerCase(),
-                  }),
+                  address: reserveData.underlyingAsset.toLowerCase(),
+                  networkId,
                 },
               ],
-              earningItems: aTokenRewardsInfo?.length
-                ? aTokenRewardsInfo.map((info) => ({
-                    amount: toDecimalNumber(
-                      info.userUnclaimedRewards,
-                      info.rewardTokenDecimals,
-                    ),
-                    label: t('earningItems.rewards'),
+              availableShortcutIds: ['deposit', 'withdraw'],
+              displayProps: {
+                title: reserveData.symbol,
+                description: `Supplied (APY: ${supplyApy.toFixed(2)}%)`,
+                imageUrl: AAVE_LOGO,
+              },
+              dataProps: {
+                manageUrl,
+                termsUrl: AAVE_TERMS_URL,
+                contractCreatedAt:
+                  AAVE_CONTRACT_CREATED_AT[
+                    getTokenId({
+                      networkId,
+                      address: reserveData.aTokenAddress.toLowerCase(),
+                    })
+                  ],
+                tvl: toSerializedDecimalNumber(
+                  toDecimalNumber(totalSupplies[i], lpTokenDecimals[i]),
+                ),
+                yieldRates: [
+                  {
+                    percentage: supplyApy,
+                    label: t('yieldRates.earningsApy'),
                     tokenId: getTokenId({
                       networkId,
-                      address: info.rewardTokenAddress.toLowerCase(),
+                      address: reserveData.underlyingAsset.toLowerCase(),
                     }),
-                  }))
-                : [],
-              depositTokenId: getTokenId({
-                networkId,
-                address: reserveData.underlyingAsset.toLowerCase(),
-              }),
-              withdrawTokenId: getTokenId({
-                networkId,
-                address: reserveData.aTokenAddress.toLowerCase(),
-              }),
-            },
-            pricePerShare: [new BigNumber(1) as DecimalNumber],
-          } satisfies AppTokenPositionDefinition),
-        // ATokens incentives
-        aTokenRewardsInfo?.length &&
-          ({
-            type: 'contract-position-definition',
-            networkId,
-            address: reserveData.aTokenAddress.toLowerCase(),
-            extraId: 'supply-incentives',
-            tokens: aTokenRewardsInfo.map((info) => ({
-              address: info.rewardTokenAddress.toLowerCase(),
+                  },
+                ],
+                earningItems: [],
```

Comment:
Are there not going to be rewards for Aave anymore? If so then this being empty makes sense but if there are rewards from the specific pool that can be pulled out that should be here. Also if there are pool specific rewards, I think there should be an item added to `yieldRates` that corresponds to the rewards APR (I think this was overlooked when yieldRates was first aded).

Reply:
There are still rewards, but to calculate the correct contribution from a given pool we need to NOT do the "shortcut" I've taken here for now.

It's what I was trying to explain in: https://github.com/valora-inc/hooks/pull/518#discussion_r1734684950

Reply:
I guess we can address this separately. The value that was returned before my changes here, was incorrect anyway.